src/sentry/integrations/github/client.py

```diff
@@ -210,6 +210,12 @@ def process_error(error: ApiError, extra: Dict[str, str]) -> None:
                 logger.warning(f"The repository is empty. {msg}", extra=extra)
             elif txt == "Not Found":
                 logger.warning(f"The app does not have access to the repo. {msg}", extra=extra)
+            elif txt == "Repository access blocked":
+                logger.warning(f"Github has blocked the repository. {msg}", extra=extra)
+            elif txt.startswith("Due to U.S. trade controls law restrictions, this GitHub"):
+                logger.warning("Github has blocked this org. We will not continue.", extra=extra)
+                # Raising the error will be handled at the task level
+                raise error
```

Comment:
The process error that this code belongs to is used when processing each repository.
If we do not raise an error, it will continue with the next repo. If we do, it will have to be handled by the task.